### ISSUE TITLE
Evaluation should use a fixed test set

### ISSUE THREAD

**Issue by marfox** [Wikidata/soweego#210]
Current behavior:
- for each (Wikidata, target) chunk pair, random split into train and test sets;
- train a model with the current training set & classify the current test set;
- compute performance evaluation.

This means we are training & testing with **different data at each iteration**, while we should always evaluate on the **same test set**.

See bb6d1e8.
